app/controllers/comments_controller.rb

```diff
@@ -1,16 +1,24 @@
 class CommentsController < ApplicationController
 
   before_action :find_commentable
+  def  specific_post
 
```

Comment:
Define empty function like this `def specific_post;end`

Reply:
resollved